metricbeat/module/mysql/performance/manifest.yml

```
@@ -0,0 +1,22 @@
+default: true
+input:
+  module: mysql
+  metricset: query
+  defaults:
+    namespace: performance
+    queries:
+      - query: >
+          SELECT digest_text, count_star, avg_timer_wait, max_timer_wait, last_seen, quantile_95
```

Btw, would be an alternative to adding dots, to explicitly put them already here?
```suggestion
          SELECT digest_text AS digest.text, count_star AS count.star, avg_timer_wait AS avg.timer.wait, max_timer_wait AS max.timer.wait, last_seen AS last.seen, quantile_95 AS quantile.95
```

If this work, this would give more control on the field names, and we wouldn't need an additional option to add dots.

---

To use dots, names need to be quoted. I have updated the previous comment to use quotes in the suggestion.

---

Yeap, this works in table response mode where you can choose the columns names. However it doesn't in variable mode where field name is a value in a `name` column. I introduced the `dedot.enabled` field to activate or deactivate it anyways.